### PR TITLE
Select to read only

### PR DESCRIPTION
**Pull request by Nase00**
* Sets `<select/>` to `readonly` to prevent annoying console error.
* Removes deprecated React.findDOMNode usage.
* Includes @tchon's dependency update.

### CODE REVIEW

**Comment by chaseadamsio** [comment]
Can you add a screenshot of the error that you see?

File: lib/select.js [hunk 1]
```diff
@@ -184,9 +185,9 @@ var classBase = React.createClass({
       this.onChange();
 
       if (!this.state.open) {
-        React.findDOMNode(this.refs['currentOption']).focus(); //eslint-disable-line dot-notation
+        this.refs['currentOption'].focus(); //eslint-disable-line dot-notation
       } else {
-          React.findDOMNode(this.refs['option' + (this.state.selectedOptionIndex || 0)]).focus();
+          ReactDOM.findDOMNode(this.refs['option' + (this.state.selectedOptionIndex || 0)]).focus();
```
Comment (Nase00) [inline lib/select.js]: I tried removing this, but it threw a `this.state.selectedOptionIndex || 0)].focus(); is not a function` error.

**Comment by chaseadamsio** [comment]
@tchon I agree. That should also be a part of linting.

**Comment by rgerstenberger** [comment]
This looks good. Nice work!

**Pull request merged**